cfme/services/service_catalogs/ssui.py

```diff
@@ -35,32 +38,63 @@ def is_displayed(self):
 
-    service_name = SSUIInput()
-    timeout = Input(name='stack_timeout')
-    db_user = Input(name="param_DBUser__protected")
-    db_root_password = Input(name='param_DBRootPassword__protected')
-    select_instance_type = BootstrapSelect("param_InstanceType")
-    stack_name = Input(name='stack_name')
-    stack_timeout = Input(name='stack_timeout')
-    resource_group = BootstrapSelect("resource_group")
-    mode = BootstrapSelect('deploy_mode')
-    vm_name = Input(name="param_virtualMachineName")
-    vm_user = Input(name='param_adminUserName')
-    vm_password = Input(name="param_adminPassword__protected")
-    vm_size = BootstrapSelect('param_virtualMachineSize')
-    user_image = BootstrapSelect("param_userImageName")
-    os_type = BootstrapSelect('param_operatingSystemType')
-    key_name = Input(name="param_KeyName")
-    ssh_location = Input(name="param_SSHLocation")
-
-    flavor = Input(name='param_flavor')
-    image = Input(name="param_image")
-    key = Input(name='param_key')
-    private_network = Input(name="param_private_network")
-    default_select_value = BootstrapSelect('service_level')
-
-    machine_credential = BootstrapSelect("credential")
-    hosts = Input(name="hosts")
+    @ParametrizedView.nested
+    class fields(ParametrizedView):  # noqa
+        PARAMETERS = ("key",)
+        input = Input(id=Parameter("key"))
+        select = Select(id=Parameter("key"))
+        param_input = Input(id=ParametrizedString("param_{key}"))
+        dropdown = VersionPick({
+            Version.lowest(): BootstrapSelect(Parameter("key")),
+            "5.9": BootstrapSelect(locator=ParametrizedString(
+                ".//div[contains(@class, 'bootstrap-select') and "
+                "select[@id={key|quote}]]"))
+        })
+        param_dropdown = VersionPick({
+            Version.lowest(): BootstrapSelect(ParametrizedString("param_{key}")),
+            "5.9": BootstrapSelect(locator=ParametrizedString(
+                ".//div[contains(@class, 'bootstrap-select') and "
+                "select[@id='param_{key}']]"))
+        })
+
+        @property
+        def visible_widget(self):
+            if self.input.is_displayed:
+                return self.input
+            elif self.dropdown.is_displayed:
+                return self.dropdown
+            elif self.param_input.is_displayed:
+                return self.param_input
+            elif self.param_dropdown.is_displayed:
+                return self.param_dropdown
+            elif self.select.is_displayed:
+                return self.select
+            else:
+                raise ItemNotFound("Visible widget is not found")
+
+        def read(self):
+            return self.visible_widget.read()
+
+        def fill(self, value):
+            return self.visible_widget.fill(value)
+
+    def fill(self, fill_data):
+        values = deflatten_dict(fill_data)
+        was_change = False
+        self.before_fill(values)
```

Comment:
this should modify was_change, there's an open issue against wt.core for this behavior as well.